src/content/topics/integrations/release.mdx

```diff
@@ -0,0 +1,99 @@
+
+## Utilizing the LaunchDarkly Integration
+
+The integration consists of two parts, both of which are automated and handled by Release.
+
+* **Pre-Deployment Step:** The LaunchDarkly integration adds a task to each of your deployments to create the feature flag environment.
+* **Pre-Delete Step:** When you delete an environment, a step is injected that cleans up by deleting the environment in LaunchDarkly.
+
+![Ephemeral Environment Creation Workflow](../../images/release-ld-integration-workflow.png)
+
+### Pre-Deployment Step
+
+The Pre-Deployment step is for when you first deploy a space or deploy a new configuration. But, it only creates the environment in LaunchDarkly the very first time it runs.
+
+![Environment &apos;ted422c&apos; was created in LaunchDarkly](../../images/release-pre-deployment-step.png)
+
+Upon successful creation, the environment variables you specified during setup are populated and exposed to your containers via K8s secrets.
+
+![LaunchDarkly Specific Envs as K8s secrets](../../images/release-ld-envs.png)
+
+You can then use these environment variables in your code to access your feature flags as you normally would. 
+
+### Pre-Delete Step
+
+The Pre-Delete step is invoked anytime a Release Environment is removed, whether that be manually through the UI or upon merge/closure of a pull request. The LaunchDarkly integration removes the corresponding LaunchDarkly environment automatically upon removal of the Release Environment.
+
+## Example: Setting up LaunchDarkly SDK Client in Rails
+
+In order to utilize LaunchDarkly in your application you need to do two things:
+
+* Authorize against LaunchDarkly with your access token
+* Configure the LD SDK client using the specific api key for your environment
+
+This example is based on this [LaunchDarkly Getting Started Guide](https://docs.launchdarkly.com/sdk/server-side/ruby#getting-started). I am also using the great [dotenv gem](https://github.com/bkeepers/dotenv) to put in default values when running locally.
+
+
+<Callout intent="info">
+  <CalloutDescription>
+    When running locally I set my LAUNCHDARKLY_ENV_SPECIFIC_SDK_KEY to the api key for my 
+    development environment in LaunchDarkly.
+  </CalloutDescription>
+</Callout>
+
+Create a file in `config/initializers/launchdarkly_client.rb`
+
+```ruby
+##retrieve access token from system ENVS or
+##local envs courtesy of dotenv
+access_token = ENV['LAUNCHDARKLY_ACCESS_TOKEN']
+
+##SETUP auth using access token
+LaunchDarklyApi.configure do |config|
+  config.api_key['Authorization'] = access_token
+  config.debugging = true
+end
+
+##set ld_client to use the api key for the specific environment
+Rails.configuration.ld_client = 
+  LaunchDarkly::LDClient.new(ENV['LAUNCHDARKLY_ENV_SPECIFIC_SDK_KEY'])
+```
+
```

Comment:
```suggestion

<CodeTabs
  defaultValue="ruby"
  values={[
    { label: 'Ruby', value: 'ruby', }
  ]
}>

    <CodeTabItem value="ruby">

    ```ruby
##Retrieve the access token from system ENVS or
##local envs with dotenv

access_token = ENV['LAUNCHDARKLY_ACCESS_TOKEN']

##Set up authorization using access token

LaunchDarklyApi.configure do |config|
  config.api_key['Authorization'] = access_token
  config.debugging = true
end

## Set ld_client to use the API key for the specific environment

Rails.configuration.ld_client = 
  LaunchDarkly::LDClient.new(ENV['LAUNCHDARKLY_ENV_SPECIFIC_SDK_KEY'])
```

    </CodeTabItem>

</CodeTabs>

```